models/public/midasnet/model.yml

```diff
@@ -44,26 +44,22 @@ files:
 framework: pytorch
 postprocessing:
   - $type: regex_replace
-    file: midas_net.py
-    pattern: 'models.'
-    replacement: ''
-  - $type: regex_replace
-    file: blocks.py
+    file: models/blocks.py
     pattern: 'align_corners=True'
     replacement: 'align_corners=False'
   - $type: regex_replace
-    file: blocks.py
+    file: models/blocks.py
     pattern: 'import torch.nn as nn'
     replacement: 'import torch.nn as nn\nimport torchvision.models as models'
   - $type: regex_replace
-    file: blocks.py
+    file: models/blocks.py
     pattern: 'torch.hub.load\("facebookresearch/WSL-Images", "resnext101_32x8d_wsl"\)'
     replacement: 'models.resnext101_32x8d()'
 conversion_to_onnx_args:
   - --model-name=MidasNet
   - --model-path=$dl_dir
   - --weights=$dl_dir/model.pt
-  - --import-module=midas_net
+  - --import-module=models.midas_net
```

Comment:
I think, it is better to set `--model-path=$dl_dir/model`. Same result but key `--model-path` more about path to source code, which is `$dl_dir/models` in that case.

Reply:
It would not be the same result, as then I wouldn't be able to remove that one post-processing step, which was the reason I did this change to begin with. :-)

Reply:
Understood